lib/ansible/parsing/vault/__init__.py

```diff
@@ -75,7 +74,7 @@
 from ansible.errors import AnsibleError
 from ansible import constants as C
 from ansible.module_utils.six import PY3, binary_type
-from ansible.module_utils.six.moves import zip
+from ansible.module_utils.six.moves import zip as _zip
```

Comment:
re  py2/py3 zip, I think that is vice-versa. On py2 six.moves.zip is itertools.izip, and on py3 it is the builtin zip.

py2.7
```
[newswoop:F25:ansible (vault_pylint %)]$ python
Python 2.7.13 (default, May 10 2017, 20:04:28) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print(zip)
<built-in function zip>
>>> print(zip.__doc__)
zip(seq1 [, seq2 [...]]) -> [(seq1[0], seq2[0] ...), (...)]

Return a list of tuples, where each tuple contains the i-th element
from each of the argument sequences.  The returned list is truncated
in length to the length of the shortest argument sequence.
>>> from ansible.module_utils.six.moves import zip
>>> print zip
<type 'itertools.izip'>
>>> print(zip.__doc__)
izip(iter1 [,iter2 [...]]) --> izip object

Return a izip object whose .next() method returns a tuple where
the i-th element comes from the i-th iterable argument.  The .next()
method continues until the shortest iterable in the argument sequence
is exhausted and then it raises StopIteration.  Works like the zip()
function but consumes less memory by returning an iterator instead of
a list.
```

py3.6
```
[newswoop:F25:ansible (vault_pylint %)]$ python3.6
Python 3.6.2 (default, Jul 24 2017, 14:22:34) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print(zip)
<class 'zip'>
>>> print(zip.__doc__)
zip(iter1 [,iter2 [...]]) --> zip object

Return a zip object whose .__next__() method returns a tuple where
the i-th element comes from the i-th iterable argument.  The .__next__()
method continues until the shortest iterable in the argument sequence
is exhausted and then it raises StopIteration.
>>> from ansible.module_utils.six.moves import zip
>>> print(zip)
<class 'zip'>
>>> print(zip.__doc__)
zip(iter1 [,iter2 [...]]) --> zip object

Return a zip object whose .__next__() method returns a tuple where
the i-th element comes from the i-th iterable argument.  The .__next__()
method continues until the shortest iterable in the argument sequence
is exhausted and then it raises StopIteration.
```